packages/Radio/src/Radio.js

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import PropTypes from "prop-types";
+import uuid from "uuid/v4";
```

Comment:
Wanna try `nanoid` instead here? (https://github.com/ai/nanoid)
Danil suggested it as a lighter weight alternative.